src/s3/commands/uploadFile.ts

```diff
@@ -265,7 +265,7 @@ async function uploadBatchOfFiles(
             location: vscode.ProgressLocation.Notification,
             title: localize(
                 'AWS.s3.uploadFile.progressTitle.batch',
-                'Uploading {0} file(s) to {1}',
+                'Uploading {0} files to {1}',
```

Comment:
When this message appears there's a lot going on, and the message changes quickly. The "file(s)" is too much visual noise and distracts from the important parts of the message.